Frontend/src/components/AdminSettings/UserProfilesContainer.jsx

```diff
@@ -1,7 +1,27 @@
 import React, { useState } from 'react';
+import './UserProfileContainer.css';
+
+import ProfileInfo from './components/ProfileInfo';
+import UserDummyData from './UserDummyData';
 
 const UserProfilesContainer = () => {
-  return <div>USER PROFILES HERE</div>;
+  const [data, setData] = useState(UserDummyData);
+  const colors = ['#81B7E3', '#C4C4C4'];
+
+  return (
+    <div className="container">
+      <button>Create Account</button>
```

Comment:
Take the "Create Account" button out of this div. The 'container' was wrapping it into a second column. Easy fix